common-content/en/module/html-css/what-is-css/index.md

```diff
@@ -34,18 +34,20 @@ selector {
 }
 ```
 
-Selectors look like this: `p { background: red }`. The selector is the `p` in this case. This `p` selects all elements of the DOM tree called `p` and sets a rule about how to show them. Selectors always come first in a ruleset.
+Rules look like this: `p { background: red; }`. The selector is the `p` in this case. This `p` selects all elements of the DOM tree called `p` and sets a rule about how to show them. Selectors always come first in a rule.
```

Comment:
```suggestion
Rulesets look like this: `p { background: red; }`. The selector is the `p` in this case. This `p` selects all elements of the DOM tree called `p` and sets a rule about how to show them. Selectors always come first in a ruleset.
```

Rule is a single declaration - this shows a ruleset containing one rule.

Reply:
Updated in [d15b267](https://github.com/CodeYourFuture/curriculum/pull/943/commits/d15b26759f625be577e9f5976d0908abe16b5127)